app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,49 @@
+class ProjectsController < ApplicationController
```

Comment:
this may not be necessary but we can use **symbols** for response status code in the entire controller just for consistency